data/org.sugarlabs.gschema.xml

```diff
@@ -94,6 +94,13 @@
                          If FALSE, Sugar will set default launch of activities in home view to 'start-new' mode.
             </description>
         </key>
+        <key name="save-as" type="b">
+            <default>false</default>
+            <summary>Resume Activity</summary>
+            <description>If TRUE, on every activity close, a Save-As popup will appear.
```

Comment:
Too verbose; there's no need to define what happens with both true and false, and it can be implied the option must be true before the behaviour will happen.